### PR TITLE
New functions to retrieve position

### PR DESCRIPTION
**Pull request by hhugo**
The goal is to improve the following pattern

```ocaml
let start, _ = Sedlexing.lexing_positions lb in
....
let _, stop = Sedlexing.lexing_positions lb in
```

### CODE REVIEW

**Review by toots — APPROVED** [review]
This looks good to me!

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# dev
+- Add API for retriving start and stop positions separatly (#155)
```
Comment (pmetzger) [inline CHANGES.md:2]: Typo. "separately".

**Review by pmetzger — COMMENTED** [review]
I have a very trivial comment.

File: CHANGES.md (hunk 1 above)
Comment (hhugo) [reply CHANGES.md]: and "retrieving"

**Comment by toots** [comment]
Thanks for fixing the typos! I think that this is good to go!

**Pull request merged**